### ISSUE TITLE
Subtotal not including tax on cart summary

### ISSUE THREAD

**Issue by cizgidekiadam** [magento/magento2#6847]
### Preconditions

1. Magento 2.1.1
### Steps to reproduce

1. Configure tax display settings to include tax in everything
### Expected result

1. Subtotal with including tax
### Actual result

1. Minicart working fine but subtotal in summary is not working
   ![image](https://cloud.githubusercontent.com/assets/4427628/19029240/1c0b60a0-894d-11e6-9be4-851a823febf3.png)
2. Shopping Cart<br/>
   ![image](https://cloud.githubusercontent.com/assets/4427628/19029204/c89c0c3a-894c-11e6-88a4-3cc599cedff7.png)
3. Backend<br/>
   ![image](https://cloud.githubusercontent.com/assets/4427628/19029219/e0c55c1c-894c-11e6-8e08-f0d09bb2e826.png)
   ![image](https://cloud.githubusercontent.com/assets/4427628/19029302/bda6a208-894d-11e6-9e20-125953bbcd7e.png)

**Comment by sevos1984** [comment]
What is your Display Product Prices In Catalog setting, and what is tax rate for your store origin and shipping address?

**Comment by sevos1984** [comment]
We are closing this issue because it hasn’t been updated in more than two weeks. If you can still reproduce this issue, please create a new GitHub issue report.

**Comment by alagukannanKumaresan** [comment]
Hello, I got first Magento project. And also version is 2.1.3. I know only Opencart & Typo3. But Magento is new for me. Is any one suggest me how can I display Subtotal or Total including Tax at Summary? I have tried the same what rbostan mentioned in admin store configurations. I am getting Sub total, but without included the Tax amount. But I am able to get Total price including Tax only in Cart, but not at Summary. Any idea? Thanks in advance!
